src/roles/foreman/tasks/main.yaml

```diff
@@ -135,6 +136,7 @@
     env:
       DYNFLOW_REDIS_URL: "redis://localhost:6379/6"
       REDIS_PROVIDER: "DYNFLOW_REDIS_URL"
+      FOREMAN_ENABLED_PLUGINS: "{{ foreman_plugins | join(' ') }}"
```

Comment:
And, is this required on Dynflow containers as well? do we enable any plugins on those containers too?